src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+public class Ball {
+    private String color;
+    private int number;
+    public void setColor(String color) {
+        this.color = color;
+    }
```

Comment:
Since you have a constructor that sets the color, this setter method might not be necessary unless there's a specific requirement for the color to be changed after the object is created.